src/components/Profile/components/PersonalInfoList/index.tsx

```diff
@@ -38,9 +38,15 @@ import CliftonStrengthsService from 'services/cliftonStrengths';
 import SLock from './Salsbury.png';
 import DPLock from './DandP.png';
 import DDLock from './DandD.png';
+import { differenceInYears, parse } from 'date-fns'; // Import a date utility library like date-fns
```

Comment:
@chanyoung-hur "like date-fns"?  It looks like it IS date-fns.  Since "import" is obvious, perhaps you mean something like 
```suggestion
import { differenceInYears, parse } from 'date-fns'; // date utility functions
```